File: 03-conditional-statements-advanced/exercises-submitted-evaluated/05.smallShop.js
function smallShop(input) {
    let item = input[0];
    let city = input[1];
    let quantity = Number(input[2]);
    let coffee = 0;
    let water = 0;
    let beer = 0;
    let sweets = 0;
    let peanuts = 0;
    let total = 0;

    switch (city) {
        case ("Sofia"):
            coffee = 0.5;
            water = 0.8;
            beer = 1.2;
            sweets = 1.45;
            peanuts = 1.6;
            break;
        case ("Plovdiv"):
            coffee = 0.4;
            water = 0.7;
            beer = 1.15;
            sweets = 1.3;
            peanuts = 1.5;
            break;
        case ("Varna"):
            coffee = 0.45;
            water = 0.7;
            beer = 1.1;
            sweets = 1.35;
            peanuts = 1.55;
            break;
    }
    switch (item) {
        case ("coffee"):
            total = coffee * quantity;
            break;
        case ("water"):
            total = water * quantity;
            break;
        case ("beer"):
            total = beer * quantity;
            break;
        case ("sweets"):
            total = sweets * quantity;
            break;
        case ("peanuts"):
            total = peanuts * quantity;
            break;
    }
    console.log(total);
}
smallShop(["coffee", "Varna", "2"]);
smallShop(["peanuts", "Plovdiv", "1"]);
smallShop(["beer", "Sofia", "2"]);
smallShop(["water", "Plovdiv", "2"]);
smallShop(["sweets", "Sofia", "2.23"]);

File: softuni-python-basics/03_conditional_statements_advanced/05_small_shop.py
product = input()
city = input()
quantity = float(input())
price_per = 0

if city == "Sofia":
    if product == "coffee":
        price_per = 0.5
    elif product == "water":
        price_per = 0.8
    elif product == "beer":
        price_per = 1.2
    elif product == "sweets":
        price_per = 1.45
    elif product == "peanuts":
        price_per = 1.6
elif city == "Plovdiv":
    if product == "coffee":
        price_per = 0.4
    elif product == "water":
        price_per = 0.7
    elif product == "beer":
        price_per = 1.15
    elif product == "sweets":
        price_per = 1.3
    elif product == "peanuts":
        price_per = 1.5
elif city == "Varna":
    if product == "coffee":
        price_per = 0.45
    elif product == "water":
        price_per = 0.7
    elif product == "beer":
        price_per = 1.1
    elif product == "sweets":
        price_per = 1.35
    elif product == "peanuts":
        price_per = 1.55

total = price_per * quantity
print(total)